src/main/java/org/webonise/gitmetrics/documents/Assignee.java

```diff
@@ -0,0 +1,14 @@
+package org.webonise.gitmetrics.documents;
+
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document
+public class Assignee {
+    public String login;
```

Comment:
you are breaking abstraction when you make the fields to be public.Please make them private!